Automattic/woocommerce-payments
# Update the UX for unsupported countries

Fixes #8367

#### Changes proposed in this Pull Request
This PR updates the Payments Connect screen, specifically enhancing the experience for merchants in locations that aren’t supported by WooPayments.

<img width="684" alt="Screenshot 2024-06-11 at 22 24 49" src="https://github.com/Automattic/woocommerce-payments/assets/8667118/151efe04-a1aa-445d-849e-ce57322d8748">
<img width="850" alt="Screenshot 2024-06-10 at 22 26 54" src="https://github.com/Automattic/woocommerce-payments/assets/8667118/8a35d07a-2aa1-440b-97c3-d1d4d47b1c09">



#### Testing instructions
- Start a fresh account onboarding process.
- Navigate to the WooCommerce settings page and change your country to India.
- Go to the Payments page.
- Verify that the WooPayments gateway is marked as not available and that several alternative payment gateways are displayed.
- From the top right corner, switch the `Business location` to a supported country.
- Confirm that a spinner preloader is displayed.
- After the loading finishes, verify that the `Welcome to WooPayments` page is displayed.
- Proceed with the onboarding process.
- Verify that the onboarding is successful with the selected `Business location`.

These steps will ensure that the experience for merchants in unsupported locations is improved and that the functionality works as expected for supported countries.

-------------------


**Post merge**
